lib/promenade/karafka/consumer_subscriber.rb

```diff
@@ -0,0 +1,44 @@
+require "promenade/karafka/subscriber"
+
+module Promenade
+  module Karafka
+    class ConsumerSubscriber < Subscriber
+      attach_to "consumer.karafka"
+
+      Promenade.histogram :kafka_consumer_batch_processing_latency do
```

Comment:
Assuming that this metric is measuring the length of time it takes to process a batch, I think `duration` is a better name than `latency` - which  has a slightly different meaning to my understanding...

🤷 it could be I just don't understand what we are measuring with this metric though :)

```suggestion
      Promenade.histogram :kafka_consumer_batch_processing_duration do
```